### PR TITLE
Fixed the issue #1206

### PR DESCRIPTION
**Pull request by saiTharunDusa**
# ✨ Pull Request

### ℹ️ About the PR

Added Masonry Layout to the app. Modified the existing components to compitable with the masonry layout.

### 🖼️ Testing Scenarios / Screenshots

Screenshot (249)
Screenshot (250)
Screenshot (253)
Screenshot (259)

### CODE REVIEW

**Comment by saiTharunDusa** [comment]
@manojVivek can you please review this pull request and give your feedback?

**Comment by manojVivek** [comment]
@violetadev Please add your review as well if you get a chance to check this one.

File: desktop-app/package.json [hunk 1]
```diff
@@ -121,17 +121,24 @@
...
     "react-detect-click-outside": "^1.1.7",
     "react-dnd": "^16.0.1",
     "react-dnd-html5-backend": "^16.0.1",
     "react-dom": "^18.2.0",
+    "react-grid-layout": "^1.4.4",
     "react-markdown": "^8.0.7",
+    "react-masonry-component": "^6.3.0",
+    "react-masonry-css": "^1.0.16",
+    "react-masonry-list": "^1.0.6",
+    "react-measure": "^2.5.2",
```
Comment (violetadev) [inline desktop-app/package.json:139]: @manojVivek we are adding a lot of new dependencies here from a third party, do we want this? this same result should be achievable without so many dependencies

Comment (manojVivek) [reply desktop-app/package.json:139]: I don't feel strongly against it, but please go ahead and add your suggestion if you feel this is unnecessary.

**Comment by saiTharunDusa** [comment]
Sure, will try to implement it using only css. If I get stuck, then I will reach on discord server. Thank you.

**Review by violetadev — APPROVED** [review]

**Pull request closed without merging**